WordPress/Jetpack/Classes/System/JetpackWindowManager.swift

```diff
@@ -17,21 +17,19 @@ class JetpackWindowManager: WindowManager {
         }
 
         guard AccountHelper.isLoggedIn else {
-            if shouldImportMigrationData {
-                importAndShowMigrationContent(blog) { [weak self] in
-                    self?.showSignInUI()
-                }
-            } else {
-                showSignInUI()
-            }
+            shouldImportMigrationData ? importAndShowMigrationContent(blog) : showSignInUI()
             return
         }
         // If the user doesn't have any blogs, but they're still logged in, log them out
         // the `logOutDefaultWordPressComAccount` method will trigger the `showSignInUI` automatically
         AccountHelper.logOutDefaultWordPressComAccount()
     }
 
-    func importAndShowMigrationContent(_ blog: Blog?, failureCompletion: (() -> ())?) {
+    func importAndShowMigrationContent(_ blog: Blog? = nil) {
+        /// Track if the WordPress installation was a migratable version
+        let wpInstallationState = MigrationAppDetection.getWordPressInstallationState()
```

Comment:
I moved this analytics call higher so that every time we're attempting to migrate data we track if the WordPress version was compatible. If you know of a better spot just let me know. 👍

Reply:
![](https://media.giphy.com/media/U2O50cAkpmTjG/giphy.gif)